unity/Assets/Scripts/BaseFPSAgentController.cs

```diff
@@ -1950,18 +1950,25 @@ protected void updateAllAgentCollidersForVisibilityCheck(bool enableColliders)
             return Time.time;
         }
 
-        private bool isSimObjVisible(Camera agentCamera, SimObjPhysics sop) {
+        private bool isSimObjVisible(Camera agentCamera, SimObjPhysics sop, bool includeInvisible, float maxDistance = 0.0f) {
```

Comment:
I refactored to always require a distance - I feel like this makes more sense since that way the isSimObjVisible can reflect the definition of what we consider visible.  I could argue for just removing the maxDistance parameters that are passed all around in the class and simply use this.maxVisibleDistance so that this method would always return the correct value.